numpy/core/tests/test_deprecations.py

```diff
@@ -400,5 +400,18 @@ def test_fortran_contiguous(self):
         self.assert_deprecated(np.ones((2,2)).T.view, args=(np.int8,))
 
 
+class TestTestDeprecated():
```

Comment:
One more tiny tweak, sorry :-). As an issue of style and avoiding weird/surprising bugs, every class should always inherit from `object`, to avoid invoking the weird old-style class infrastructure on python 2.

Let me know when you've had a chance to fix this and I'll merge :-)